README.md

```diff
@@ -69,18 +81,19 @@ npm i
 
 To run this application locally, the first step is to create a local .env file and set the environment variables with the appropriate values. You can duplicate .env.example file and rename it to .env
 
+_Please note, in order run this project with your local database, you need an existing postgres username, and password. Some machines will have a superuser by default, when installed, but there may be cases where you need to create a role for postgres_.
+
```

Comment:
This is fine for now since I don't want you to spend anymore time on it, but as you wrestled with this problem did you ever try anything like this? 

```
if ! psql -c "SELECT usename FROM pg_catalog.pg_user" | grep -q $DB_USER; then
    psql -c "CREATE USER $DB_USER WITH LOGIN INHERIT"
fi

psql -c "ALTER USER $DB_USER WITH PASSWORD '$DB_PASSWORD'"
psql -c "GRANT CREATE ON DATABASE $DB_NAME, $TEST_DB_NAME TO $DB_USER" 
```

Reply:
No I haven't. I thought we can only run standard SQL queries in a script file! Good to know!

Reply:
@klavenjones